pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -231,6 +257,46 @@ func (c *Cluster) startProvisioning(config *provisionConfig) {
 		return
 	}
 
+	for _, volume := range c.ValidStorage.VolumeSources {
+		osdProps := osdProperties{
+			crushHostname: volume.PersistentVolumeClaimSource.ClaimName,
+			pvc:           volume.PersistentVolumeClaimSource,
+			resources:     volume.Resources,
+			placement:     volume.Placement,
+		}
+
+		// update the orchestration status of this pvc to the starting state
+		status := OrchestrationStatus{Status: OrchestrationStatusStarting, PvcBackedOSD: true}
+		if err := c.updateNodeStatus(osdProps.crushHostname, status); err != nil {
+			config.addError("failed to set orchestration starting status for pvc %s: %+v", osdProps.crushHostname, err)
+			continue
+		}
+
+		job, err := c.makeJob(osdProps)
+		if err != nil {
+			message := fmt.Sprintf("failed to create prepare job for pvc %s: %v", osdProps.crushHostname, err)
+			config.addError(message)
+			status := OrchestrationStatus{Status: OrchestrationStatusCompleted, Message: message, PvcBackedOSD: true}
+			if err := c.updateNodeStatus(osdProps.crushHostname, status); err != nil {
+				config.addError("failed to update pvc %s status. %+v", osdProps.crushHostname, err)
+				continue
+			}
+		}
+
+		if _, err := c.context.Clientset.BatchV1().Jobs(job.Namespace).Create(job); err != nil {
```

Comment:
if `errors.IsAlreadyExists(err)`, this is expected during a later orchestration, right? We will just skip ever provisioning the job again.

Reply:
cc @ashishranjan738